nirc_ehr/resources/queries/study/casesTemp.js

```diff
@@ -0,0 +1,67 @@
+
+var console = require("console");
+var LABKEY = require("labkey");
+
+var prevRow;
+
+function init() {
```

Comment:
Is this here because we lose the prevRow value when we reach the end of a batch (and commit the transaction and perhaps throw away the JS context/scope)? A comment would be handy.

Reply:
There was a comment right below that but I elaborated more on it